install.c

```diff
@@ -4,26 +4,73 @@
 #include "open.h"
 #include "readwrite.h"
 #include "exit.h"
+#include "alloc.h"
+#include "stralloc.h"
 
 extern void hier();
+extern char *getenv(const char *name);
```

Comment:
In the qmail codebase, getenv is env_get, requiring the header env.h and linking against env.a.

Reply:
Good catch. It was also missing fifo.h (DJBs fault, he deleted it in 1.03) and str.h (for str_len(), my fault).